### PR TITLE
Avoid using specific environment uuid variables in job definition

### PR DESCRIPTION
**Pull request by liu-sheng**

### CODE REVIEW

File: playbooks/gophercloud-telefonica-acceptance-test/run.yaml [hunk 1]
```diff
@@ -18,11 +20,20 @@
...
+          if [ -z "$OS_IMAGE_ID" ]; then
+              curl -O http://download.cirros-cloud.net/0.3.5/cirros-0.3.5-x86_64-disk.img
+              export OS_IMAGE_ID="$(openstack image create --file ./cirros-0.3.5-x86_64-disk.img --min-disk 1 --container-format bare --disk-format raw cirros-0.3.5-x86_64-disk -f value -c id)"
+              rm cirros-0.3.5-x86_64-disk.img
+          fi
+          export OS_NETWORK_NAME="gophercloud-telefonica-test"
+          export OS_NETWORK_ID="$(openstack network show $OS_NETWORK_NAME -f value -c id)"
+          if [ -z "$OS_NETWORK_ID" ]; then
+              export OS_NETWORK_ID="$(openstack network create $OS_NETWORK_NAME -f value -c id)"
+              openstack subnet create --network $OS_NETWORK_ID --subnet-range 10.0.10.0/24 gophercloud-telefonica-test
```
Comment (animationzl) [inline playbooks/gophercloud-telefonica-acceptance-test/run.yaml]: replace gophercloud-telefonica-test with $OS_NETWORK_NAME for consistence

Comment (liu-sheng) [reply playbooks/gophercloud-telefonica-acceptance-test/run.yaml]: thanks for review!, actually they are different things, one is network name, another is subnet name, even I make them same. will update this patch by defining a local variable for subnet name.

**Comment by kiwik** [comment]
LGTM, thank you.

**Comment by kiwik** [comment]
Approved

**Pull request merged**